modules/sql-engine/src/testFixtures/java/org/apache/ignite/internal/sql/engine/util/QueryChecker.java

```diff
@@ -294,6 +294,8 @@ static Matcher<String> containsAnyScan(String schema, String tblName, String...
 
     QueryChecker returns(Object... res);
 
+    QueryChecker returnMatched(Matcher<Iterable<?>> matcher);
```

Comment:
May be rename it to `returns`? :thinking:

Reply:
```
QueryChecker returns(Object... row);
QueryChecker returns(Matcher<Iterable<?>> rowMatcher);
```